Code/Core/Math/Random.cpp

```diff
@@ -24,4 +26,15 @@ uint32_t Random::GetRand()
     return( (uint32_t)( m_Seed / ( ( CORE_RAND_MAX + 1 ) * 2 ) ) % ( CORE_RAND_MAX + 1 ) );
 }
 
+// GetRandString
+//------------------------------------------------------------------------------
```

Comment:
I don't think this function belongs in the Random class.
It formats a random number in a way that is likely not very general purpose and can be trivially implemented with a single line in the calling code (as shown below) or with 2 lines if you need the "randNum".

Reply:
I deleted the entire function